### ISSUE TITLE
Save to audio.com dialog: missing accessibility names

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#6196]
### Steps to reproduce

With a screen reader running, tab around the dialog

### Expected behavior

When in "person icon" is the focus, the screen reader reads either "account not linked" or the user name.
When the project name text box is the focus, the screen reader reads "project name" as the name of the text box

### Actual behavior

The accessibility names are both empty.

### Audacity Version

latest beta

### Operating system

Windows

**Comment by DavidBailes** [comment]
The accessibility name of the "person" icon was fixed by https://github.com/audacity/audacity/pull/6211

**Comment by LWinterberg** [comment]
ah, wait, but "When the project name text box is the focus, the screen reader reads "project name" as the name of the text box" presumably wasn't
